tasks/managed/collect-atlas-params/README.md

```diff
@@ -19,6 +19,9 @@ strings as results, indicating that the Atlas push should be skipped.
 | taskGitUrl              | The url to the git repo where the release-service-catalog tasks and stepactions to be used are stored                      | No          | ""                      |
 | taskGitRevision         | The revision in the taskGitUrl repo to be used                                                                             | No          | ""                      |
 
+## Changes in 1.2.0
+* Added compute resource limits
```

Comment:
OK, but why. The values look pretty small - so is that the goal - to reduce the resources for these, because they don't need much?

Reply:
I have the same question. But the test passed. Seems it's enough.

Reply:
Yes, I am merely asking for some reasoning. OK, if I go to the Jira, it's immediately clear:

> The tasks at https://github.com/redhat-appstudio/release-service-bundles/tree/main/catalog/task need to have limits set for them to prevent them from requesting too many resources.

But there is no reason we couldn't have some basic reasoning here - at the very least in the commit message and PR description.

Reply:
Because the ticket is to add computeResources and these tasks simply echo a string into a result so I would guess they don't need much

Reply:
> Yes, I am merely asking for some reasoning. OK, if I go to the Jira, it's immediately clear:
> 
> > The tasks at https://github.com/redhat-appstudio/release-service-bundles/tree/main/catalog/task need to have limits set for them to prevent them from requesting too many resources.
> 
> But there is no reason we couldn't have some basic reasoning here - at the very least in the commit message and PR description.

I think that is super obvious. What is the alternative? We are adding resource limits to ensure that the tasks take more than they need and hog resources? Of course we are adding limits to prevent the tasks from taking too many

Reply:
> I think that is super obvious. 

No, without the context it's not obvious at all. The most typical reason why you would add a limit is because something failed on low resources. It totally makes sense to limit it here to avoid wasting resources. But without any explanation, how is the reviewer supposed to know that's the goal here?